jest.config.base.js

```diff
@@ -1,13 +1,16 @@
-module.exports = {
-  preset: 'ts-jest',
+const tsJestConfig = {
+  tsconfig: 'tsconfig.settings.json',
+  isolatedModules: true
+}
+module.exports.tsJestConfig = tsJestConfig
```

Comment:
**praise**: nice, this was a sticking point when i last tried this upgrade, nice way of solving it